docs/03-endpoints/api-admin/introduction.md

```diff
@@ -5,6 +5,8 @@
 
 # Introduction: Using the Admin API
 
+The DSP Admin API makes it possible to administer projects, users, user groups, permissions, and hierarchical lists.
```

Comment:
```suggestion
The DSP Admin API makes it possible to administrate projects, users, user groups, permissions, and hierarchical lists.
```

sounds less linke "giving medicine" in my ears

Reply:
;) thanks, there were also other locations where this strange word was used. I updated all of them.